.github/workflows/analyze.yaml

```diff
@@ -1,12 +1,26 @@
 #
-# Repository validation
+# Repository analysis
 #
+
+# NOTES:
+# This worflow uses PSRule, CodeQL, and DevSkim.
+# You can read more about these linting tools and configuration options here:
+#   PSRule - https://aka.ms/ps-rule and https://github.com/Microsoft/PSRule.Rules.MSFT.OSS
+#   CodeQL - https://codeql.github.com/docs/codeql-overview/about-codeql/
+#   DevSkim - https://github.com/microsoft/DevSkim-Action and https://github.com/Microsoft/DevSkim
+
 name: Analyze
 on:
-- pull_request
+  push:
+    branches: [ main, 'release/*' ]
+  pull_request:
+    branches: [ main, 'release/*' ]
+  schedule:
+    - cron: '26 21 * * 0'
```

Comment:
Lets add a comment like `# At 09:26 PM, only on Sunday` to indicate what this cron expression does in english.